.github/workflows/ci.yml

```
@@ -1,10 +1,20 @@
 name: CI
 
-on: [push]
+on:
+  pull_request:
+  push:
+    branches: [main]
```

```suggestion
on:
  workflow_dispatch:
  pull_request:
  push:
    branches: [main]
```

unrelated, but can we also add this, so the CI can be manually triggered?

---

done